src/components/structures/RoomView.js

```diff
@@ -428,6 +429,12 @@ module.exports = React.createClass({
                 roomView.addEventListener('dragend', this.onDragLeaveOrEnd);
             }
         }
+        if (this.refs.messagePanel && !this.state.atEndOfLiveTimelineInit) {
```

Comment:
can't we do this in componentDidMount instead?

Reply:
The ref is undefined at that point still, so we try and catch it when it first gets defined.

Reply:
Hmm, https://reactjs.org/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element says:
> ref updates happen before componentDidMount or componentDidUpdate lifecycle methods.

Reply:
That's the theory, but I was distinctly getting undefined there. It might be because of the conditions we have in `render` which allow for other things to render instead of the panel on first load, even if only briefly.

Reply:
:cry: ok then...

Reply:
Maybe at a comment with the above, so nobody tries to move it to componentDidMount in the future without proper testing?